Cargo.toml

```diff
@@ -28,13 +28,17 @@ time = { version = "0.3.11", features = ["formatting", "local-offset", "parsing"
 ## Use `libgit2` as a backend for git operations
 git2 = { version = "0.16.0", default-features = false, optional = true }
 
+## Rust alternative to `libgit2`
+gix = { version = "0.38.0", optional = true }
```

Comment:
In more recent versions, compile times have been reduced. Generally, improving compile times with the right set of feature flags (which might not exist yet) should help crates like these to keep compile times minimal. Otherwise it will always be a point of criticism that can be anticipated.